lib/src/lib.rs

```diff
@@ -46,10 +46,10 @@ pub fn create_zingoconf_with_datadir(
         let config = ZingoConfig {
             server_uri: Arc::new(RwLock::new(server)),
             chain: match info.chain_name.as_str() {
```

Comment:
and I think here as well...

Reply:
Right...  I thought that you wanted us to match the interface of `info`?

Reply:
yes, match the interface of `info` (chain_name) and with the new feature `parse` (network)... both with the same name and the same values....

Reply:
It looks like the `InfoCommand` will return an object with:

https://github.com/zancas/zingolib/blob/90996dc2862249d81af6bf3013a1bc6ce4278bdb/lib/src/lightclient.rs#L585

`"chain_name" => i.chain_name,`

Meanwhile will return an object with a key `chain_name` like this:

https://github.com/zancas/zingolib/blob/90996dc2862249d81af6bf3013a1bc6ce4278bdb/lib/src/commands.rs#L98

"chain_name" => chain_name_string,


So the interface to each should see a key "chain_name"..  right?

Reply:
yes, sounds perfect.